ci/build-push-x86_64.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+source "$(dirname "${BASH_SOURCE[0]}")/variables.sh"
+
+docker buildx build \
+    --push \
+    --platform=linux/amd64 \
+    --tag "${REPO_URL}:${BUILD_TAG}-${amd64_arch}" \
+    --file "Dockerfile.${amd64_arch}" \
```

Comment:
Where are variables like `amd64_arch` coming from?

Reply:
From the `variables.sh` script that is being sourced a couple lines above.